cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -229,6 +233,26 @@ public ScheduleState scheduleState(ProgramId program, SchedulableProgramType pro
     }
   }
 
+  @Nullable
+  public static String removeAppVersion(String scheduleId) {
```

Comment:
Also, I don't like the idea of having public static method (which essentially is a util method).  Why it is public static?